src/js/connection.js

```diff
@@ -0,0 +1,256 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+
+(function (fluid) {
+    "use strict";
+    var youme = fluid.registerNamespace("youme");
+
+    /**
+     *
+     * A MIDI Connection represents a connection to a single input or output port. Unless you are completing the wiring
+     * yourself, you should not use this directly, but should use `youme.connection.input` or `youme.connection.output`
+     * instead.
+     *
+     **/
+    fluid.defaults("youme.connection", {
+        gradeNames: ["fluid.modelComponent"],
+
+        model: {
+            port: false,
```

Comment:
Some type instability here in that it seems that when open this holds a MIDIPort - probably null is a better initial value, and also document this type in the model

Reply:
I added a note here and updated the initial value.